src/dotnet/APIView/APIViewWeb/ExtensionMethods.cs

```diff
@@ -9,4 +12,41 @@ public static string GetGitHubLogin(this ClaimsPrincipal user)
             return user.FindFirst(c => c.Type == "urn:github:login")?.Value;
         }
     }
+    public class OrganizationRequirement: IAuthorizationRequirement
```

Comment:
1 type per file

Reply:
Should I put both extension methods in a generic extension method class in this file?

Reply:
Nah, they are unrelated. But you can remove this extension method altogether, it has only one usage.